src/components/organisms/AssetActions/Trade/Output.tsx

```diff
@@ -0,0 +1,67 @@
+import { useOcean } from '@oceanprotocol/react'
+import { FormikContextType, useFormikContext } from 'formik'
+import React, { ReactElement, useEffect, useState } from 'react'
+import { TradeLiquidity } from '.'
+import Token from '../Pool/Token'
+import styles from './Output.module.css'
+
+export default function Output({
+  dtSymbol,
+  poolAddress
+}: {
+  dtSymbol: string
+  poolAddress: string
+}): ReactElement {
+  const { ocean } = useOcean()
+  const [maxOutput, setMaxOutput] = useState<string>()
+  const [swapFee, setSwapFee] = useState<string>()
+
+  // Connect with form
+  const { values }: FormikContextType<TradeLiquidity> = useFormikContext()
+
+  // Get swap fee
+  useEffect(() => {
+    if (!ocean || !poolAddress) return
+
+    async function getSwapFee() {
+      const swapFee = await ocean.pool.getSwapFee(poolAddress)
+      // swapFee is tricky: to get 0.1% you need to convert from 0.001
+      setSwapFee(`${Number(swapFee) * 100}`)
+    }
+    getSwapFee()
+  }, [ocean, poolAddress])
+
+  // Get output values
+  useEffect(() => {
+    if (!ocean || !poolAddress) return
+
+    async function getOutput() {
+      // Minimum received
+      // TODO: check ity is actually the minimum received after transaction.
+      const maxPrice =
+        values.type === 'buy'
+          ? await ocean.pool.getOceanNeeded(poolAddress, `${values.ocean}`)
+          : await ocean.pool.getDTNeeded(poolAddress, `${values.datatoken}`)
```

Comment:
the output values for this are kinda weird, had to remove that `1` param in `ocean.pool.getDTNeeded` cause the typings want something else.

With this change I'm getting this output, which looks wrong:

<img width="476" alt="Screen Shot 2020-11-10 at 16 03 47" src="https://user-images.githubusercontent.com/90316/98691440-52f38d80-236e-11eb-93f5-999652e7da02.png">

<img width="494" alt="Screen Shot 2020-11-10 at 16 00 57" src="https://user-images.githubusercontent.com/90316/98691373-42431780-236e-11eb-9cda-4df79da1b7e7.png">

Reply:
Oh, sorry I totally ignored those values because we said that we remove that section. Are getting it back? It was unclear to me